### PR TITLE
Added ability to hide gutter and time indicator

### PR DESCRIPTION
**Pull request by paustint**
Added the ability to pass in two additional optional props  and
hideGutter=true will hide the gutter in an view where there is a time sidebar (except agenda view)
hideTimeIndicator=true will hide the time indicator in views where this is shown

If the gutter is hidden, then the time indicator will also be hidden

Impacted Views:
**week**, **work_week**, **day**

image

image

### CODE REVIEW

**Comment by jquense** [comment]
Sorry for the delay :P This looks good can you update and rebase on the current master?

File: src/TimeGrid.js [hunk 1]
```diff
@@ -188,15 +193,17 @@ export default class TimeGrid extends Component {
...
-          <div ref="timeIndicator" className="rbc-current-time-indicator" />
-
-          <TimeColumn
-            {...this.props}
-            showLabels
-            style={{ width }}
-            ref={gutterRef}
-            className="rbc-time-gutter"
-          />
+          {!hideTimeIndicator && <div ref="timeIndicator" className="rbc-current-time-indicator" />}
```
Comment (jquense) [inline src/TimeGrid.js]: If we aren't showing the indicator we should also not be calculating it's position on an interval. Can you also disable that when this is true?

**Comment by paustint** [comment]
@jquense - sorry, I have been on vacation for a bit. I am going to close this PR since there has been a lot of other changes since this was first opened and I need to get up to speed on those other changes and will revisit the changes in this PR and probably open another PR with your suggested changes as well.

**Comment by snehjogani** [comment]
i'm unable to hide the time indicator by passing the above mentioned options. 

```
<BigCalendar
            localizer={localizer}
            events={set}
            popup
            step={60}
            hideTimeIndicator
            showMultiDayTimes
            defaultDate={new Date()}
            views={{ month: true, week: true, day: true }}
            startAccessor={'startDate'}
            endAccessor={'endDate'}
            tooltipAccessor={'status'}
            eventPropGetter={event => ({
              style: { backgroundColor: event.colorCode },
            })}
            onSelectEvent={({ id }) => history.push(`Dispatches/${id}`)}
          />
```

Remark if i'm doing something wrong here.

**Comment by asashay** [comment]
would be awesome to have this one in the lib :)
@paustint do you still plan to add it?

**Comment by iamgbayer** [comment]
Some update about this feature?

**Comment by angela318** [comment]
This is not merged?

**Comment by AndresFSalcedo** [comment]
Hi guys,
Any update on this matter?

**Comment by craftzneko** [comment]
@paustint @jquense  was this ever merged assume it was not, has there been later work to do this?, would be very useful for all day events. Have any others found a good solution @AndresFSalcedo @angela318  @iamgbayer @asashay @snehjogani

**Comment by cutterbl** [comment]
@craftzneko I don't think anyone ever updated/recreated the PR with current code, so it never made it into master.

**Comment by craftzneko** [comment]
Is there any hope for this feature to be added, I am using a plugin for grafana that has this as part of their code and hiding the time is critical in my use case as it doesnt make sense having times listed that will confuse users. If this just is not going to happen is there a way i could make it work on my side @paustint dont have much experience with this codebase but i can take instruction if someone with more knowledge than me can explain how to achieve

**Comment by cutterbl** [comment]
@craftzneko Someone would need to recreate the PR with current code

**Pull request closed without merging**